src/pages/registration/description_tab/vocabularies/VocabularyFields.tsx

```diff
@@ -102,7 +97,7 @@ export const VocabularyFields = ({ defaultVocabularies, allowedVocabularies }: V
                   {!defaultVocabularyKeys.includes(vocabulary) && (
                     <>
                       <Button
-                        sx={{ height: 'fit-content', alignSelf: 'center' }}
+                        sx={{ ml: '2rem', minWidth: 'max-content' }}
```

Comment:
`gap: '0.5rem 1rem'` på flex i stedet for `ml` her sånn at det blir aligned på mobil også?

```suggestion
                        sx={{ minWidth: 'max-content' }}
```

Reply:
Ah, overså den! Fjerner margin på `xs` skjerm, for å bevare spacing på stor skjerm.